src/rpm/package.rs

```diff
@@ -540,6 +548,64 @@ impl PackageMetadata {
         }
     }
 
+    fn get_scriptlet(
+        &self,
+        scriptlet_tag: IndexTag,
+        flags_tag: IndexTag,
+        program_tag: IndexTag,
+    ) -> Result<Scriptlet, Error> {
+        let script = self.header.get_entry_data_as_string(scriptlet_tag);
+        let flags = self.header.get_entry_data_as_u32_array(flags_tag);
```

Comment:
The documentation and code comments imply that this ought to be a lone integer rather than an integer array, although I know that some fields are "fuzzy" in this respect.  What is the motivation for treating it as an array of integers?

Reply:
@dralley oops, I was following along with Dependency's implementation and I noticed that it used u32_array for it's flags field and is also uses a u32 bitflag so I just thought that was the right option. I changed it to *_u32 and seems to still work.